### ISSUE TITLE
How to translate .md files into other languages

### ISSUE THREAD

**Issue by lim4349** [Okazari/Rythm.js#102]
Translating .md files into other languages may attract programmers from around the world.

**Comment by Okazari** [comment]
You're right, i also feel that developper needs to be able to read a documentation in english ;) (personnal opinion.)
I can't adress this issue by myself, but anyone can feel free to create a PR on that subject. (like #104)
Maybe could you add a point about this in `Contributing.md` ?
